docs/integrations/kubeflow.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: Kubeflow
+description: Easily build reproducible data pipelines with Kubeflow and lakeFS using commits, without modifying the code or logic of your job. 
+parent: Integrations
+nav_order: 56
+has_children: false
+
+---
+
+
+[Kubeflow](https://www.kubeflow.org/docs/about/kubeflow/) is a project dedicated to making deployments of ML workflows on Kubernetes simple, portable and scalable.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC 
+{:toc}
+
+## Using lakeFS with Kubeflow pipelines
+{: .no_toc }
+A Kubeflow pipeline is a portable and scalable definition of an ML workflow composed of steps. Each step in the pipeline is an instance of a component represented as an instance of [ContainerOp](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html#kfp.dsl.ContainerOp).
+
+There are some aspects we will need to handle to run Kubeflow with lakeFS:
+## Install lakeFS CLI 
```

Comment:
```suggestion
## Prerequisites
### Install lakeFS CLI 
```

Reply:
Recommend to use a blank line between headers and content - or any markdown block.